packages/framework/src/Component/Doctrine/Exception/DuplicatedAliasException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Doctrine\Exception;
+
+use LogicException;
+use Throwable;
+
+class DuplicatedAliasException extends LogicException
+{
+    /**
+     * @param string $alias
+     * @param int|null $code
+     * @param \Throwable|null $previous
+     */
+    public function __construct(string $alias, ?int $code = 0, ?Throwable $previous = null)
+    {
+        $message = sprintf(
+            'Alias "%s" is already assigned to different entity.',
```

Comment:
I'm thinking about a more verbose message, so it's really clear what's going on. 
What do you think of 

```php
sprintf('You cannot use "%s" as "%s" because this alias is already assigned to "%s" in the same QueryBuilder instance.',
    $resolvedClass,
    $alias,
    $resolvedJoinClass
);
```

`You cannot use "App\Model\Payment\Payment" as "p" because this alias is already assigned to "App\Model\Product\Product" in the same QueryBuilder instance.`

Reply:
Nice point. I gues it will be the most descriptive exception in FW 😄